lib/ansible/modules/database/mysql/mysql_db.py

```diff
@@ -297,16 +315,20 @@ def main():
     single_transaction = module.params["single_transaction"]
     quick = module.params["quick"]
 
+    if len(db) > 1 and state != 'dump':
```

Comment:
Are you somewhere making sure that `db` is not an empty list?

Reply:
+1 Felixfontein

Fixed this